src/cli/deploy.ts

```diff
@@ -52,4 +54,8 @@ export default function (program: RootCmd) {
         process.exit(1);
       }
     });
+
+  // Wait for the resources to be fully up
+  const replicasReady = await checkAllDeploymentReplicas();
+  Log.info(`All replicas ready: ${replicasReady}`);
```

Comment:
we don't need this, it is already covered in another function

Reply:
Removed log line